docs/ml/deploy.md

```diff
@@ -150,9 +150,9 @@ The model package config looks like this:
 
 ### `tflite_cpu` Limitations
 
-We strongly recommend that you package your `.tflite_cpu` model with metadata in [the standard form](https://github.com/tensorflow/tflite-support/blob/560bc055c2f11772f803916cb9ca23236a80bf9d/tensorflow_lite_support/metadata/metadata_schema.fbs).
+We strongly recommend that you package your `tflite_cpu` model with metadata in [the standard form](https://github.com/tensorflow/tflite-support/blob/560bc055c2f11772f803916cb9ca23236a80bf9d/tensorflow_lite_support/metadata/metadata_schema.fbs).
 
-In the absence of metadata, your `.tflite_cpu` model must satisfy the following requirements:
+In the absence of metadata, your `tflite_cpu` model must satisfy the following requirements:
```

Comment:
drive by fixes. This context is the model name, not the model's definition file extension, which is `.tflite`